Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM nvidia/cuda:12.1.0-devel-ubuntu22.04 as base
+RUN apt-get update && \  
+    apt-get upgrade -y 
+
+RUN apt-get update && \  
+    # apt-get upgrade -y && \
+    apt-get install -y python3.10 python3-pip git wget && \  
+    apt-get autoremove -y && \  
+    apt-get clean
+
+# install ffmpeg
+RUN wget https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-amd64-static.tar.xz &&\
+    wget https://johnvansickle.com/ffmpeg/builds/ffmpeg-git-amd64-static.tar.xz.md5 &&\
+    md5sum -c ffmpeg-git-amd64-static.tar.xz.md5 &&\
+    tar xvf ffmpeg-git-amd64-static.tar.xz &&\
+    mv ffmpeg-git-*-static/ffprobe ffmpeg-git-*-static/ffmpeg /usr/local/bin/ &&\
+    rm -rf ffmpeg-git-*
```

Comment:
In my Dockerfile, I used ffmpeg from the existing Ubuntu package sources and everything seemed to work well. Any reason this custom version should be installed?

This goes more to the maintainers I guess.

Reply:
Yeah, on our prod server, we were running ffmpeg that came with Amazon Linux’s Deep Learning AMI. It turned out that ffmpeg had a bug and was cutting audio short during mp3->wav conversions, so I added this just to be safe given prior scars :)

Reply:
If not necessary, you may want to remove ffprobe if not used to save space from the static build.